### PR TITLE
rulefit demo [nocheck]

### PR DESCRIPTION
**Pull request by koniecsveta**

### CODE REVIEW

File: h2o-py/demos/rulefit_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,1544 @@
...
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import h2o\n",
+    "from h2o.estimators import H2ORuleFitEstimator, H2ORandomForestEstimator\n",
+    "\n",
+    "# init h2o cluster\n",
+    "h2o.init(strict_version_check=False, url=\"http://192.168.59.147:54321\")"
```
Comment (honzasterba) [inline h2o-py/demos/rulefit_demo.ipynb:113]: would keep just `h2o.init()`

**Review by honzasterba — APPROVED** [review]
minor comment
what about adding an example flow as well?

**Comment by honzasterba** [comment]
this should probably go to release branch

**Comment by koniecsveta** [comment]
Hm I've noticed a problem with displayment of long rules, they are displayed like this:
`(sex in {female}) & (sibsp < 3.5 or sibsp is NA) & (pclass in {1, ...`
hence not readable..  checked also in flow and there the displayment is ok...

**Comment by michalkurka** [comment]
Comparing the model performance can be tricky but for the purposes of this demo (which is understood to be just an illustration of the concept) you should use the test dataset (that you don't use in any part of the training) instead of performance on the training frame.

**Comment by koniecsveta** [comment]
will remove this comparison

**Review by michalkurka — APPROVED** [review]
thank you

**Pull request merged**